src/widget/wtrackmenu.h

```diff
@@ -37,22 +37,14 @@ class WTrackMenu : public QMenu {
         HideUnhidePurge = 1 << 9,
         FileBrowser = 1 << 10,
         Properties = 1 << 11,
-        IndependentFilters = AutoDJ |
-                Playlist |
-                Crate |
-                Metadata |
-                Reset |
-                BPM |
-                Color |
-                FileBrowser |
-                Properties
+        TrackModelFilters = LoadTo | Remove | HideUnhidePurge
```

Comment:
LoadTo doesn't seem to depend on a TrackModel*.

Reply:
LoadTo depends on the client to emit the load track signal. Currently, only a WTrackTableView does so.

Reply:
That's not the same as depending on having a TrackModel. It is only a coincidence that the only place we currently use WTrackMenu with LoadTo, we also have a TrackModel.

Reply:
Cool. I'll remove LoadTo from this list.

Reply:
It would help to add a comment here that the `loadTrackToPlayer` signal must be handled by the code that uses WTrackMenu for the LoadTo feature to work.